reactioncommerce/reaction
# refactor: use npm package for carts plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6168

This PR removes the internal `carts` plugin, and installs the new `api-plugin-carts` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `carts` is the correct version in the `systemInformation` query
- Start Reaction, add a product to cart, checkout, etc

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
